### ISSUE TITLE
az bicep install freezes

### ISSUE THREAD

**Issue by michaelsrichter** [Azure/azure-cli#25330]
## Describe the bug

**Command Name**
`az bicep install`

Freezes here:

**Errors:**
```
Installing Bicep CLI v0.14.6...
```

It looks like the binary does get installed but when I try to run `az bicep version` I got an error message:
```
PermissionError: [Errno 13] Permission denied: '/home/mike/.azure/bin/bicep'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az bicep install`

## Expected Behavior

it should all work

## Environment Summary
```
Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.31, Ubuntu 20.04.5 LTS
Python 3.10.8
Installer: DEB

azure-cli 2.44.1
```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@michaelsrichter Thanks for reaching out to us and reporting this issue. We are looking into this issue and we will provide an update.

**Comment by michaelsrichter** [comment]
Thanks for the assistance on this. I initially thought it was an ownership issue as well. I ran a similar command but then I got a different error. I forgot the details of that error but it made me think there was something wrong with the `bicep` executable itself. 

When I downloaded and installed Bicep directly via the instructions here: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install - this Bicep executable worked (the one hosted here: `/usr/local/bin/bicep`).

But this did not fix the `az bicep` issue.

I DID solve this for myself by copying the `bicep` executable from `/usr/local/bin/bicep` to `/home/mike/.azure/bicep` and now the `az bicep` command works and I am unblocked.
